fecgov/fec-eregs
# Add node-sass package to dev dependencies list

## Summary (required)

- Resolves #644

- Move node-sass@7.0.1 to dev dependencies as this package is required only during development.


### Required reviewers

one developer

## How to test

(Include any information that may be helpful to the reviewer(s). This might include links to sample pages to test or any local environmental setup that is unusual such as environment variable (never credentials), API version to point to, etc)

- checkout branch
- rm -rf node_modules
- npm i
- npm run build
- python manage.py compile_frontend
- echo API_BASE = 'https://fec-prod-eregs.app.cloud.gov/regulations/api/'>> local_settings.py (Point to Prod Eregs API. An alternative if 2021 regs are not parsed onto your local eregs-db)
- python manage.py runserver
- http://localhost:8000/ (make sure the style sheets render OK on the browser)
